src/app/about/about.component.html

```diff
@@ -0,0 +1,263 @@
+          </p>
+          <h5>How to Cite Dockstore</h5>
+          <ul>
+            <li>
+              For citing Dockstore as a paper, take a look at our
+              <a target="_blank" rel="noopener noreferrer" href="https://f1000research.com/articles/6-52/v1">F1000 paper</a>.
+            </li>
+            <li>
+              For citing the actual code, take a look at our
+              <a target="_blank" rel="noopener noreferrer" href="https://zenodo.org/record/4589367#.YGSZyRNKi3I">Zenodo entry</a>.
+            </li>
+          </ul>
+          <h5>Future Plans</h5>
+          <p>
+            We plan on expanding Dockstore in several ways over the coming months. For transparency, we maintain a list of long term plans
+            for larger features going forward.
+          </p>
+          <a [href]="Dockstore.DOCKSTORE_REPO + '/wiki/Dockstore-Roadmap'" target="_blank" rel="noopener noreferrer">
+            <button class="btn">Dockstore Roadmap</button>
+          </a>
+        </div>
+        <div fxLayout="column" fxFlex>
+          <mat-tab-group mat-align-tabs="center" class="workflow-tab-group">
+            <mat-tab label="COVID-19">
+              <mat-card class="mat-elevation-z4 about-card">
+                <div fxLayout="row" fxLayoutAlign="space-around start">
+                  <div fxLayout="column" fxLayoutAlign="space-between start">
+                    <div fxLayout="row" fxLayout.lt-sm="column" fxLayoutGap="2rem">
+                      <div fxLayout="column" fxLayoutAlign="start start" fxLayoutGap="1rem" fxLayout.lt-sm="row">
+                        <img src="../../assets/images/sponsors/coloured/broad.svg" alt="Broad Image" />
+                        <span class="workflowlength">{{ workflowsLength$ | async }} workflows</span>
+                      </div>
+                      <span>
+                        The <strong>Broad Viral Genomics & Data Sciences Platform</strong> hit the ground running, producing a collection of
+                        <strong>viral NGS workflows for analyzing COVID-19</strong>. Learn how to use them with your own data in linked
+                        Terra workspaces.
+                      </span>
+                    </div>
+                    <span><strong>Workflows functions include:</strong></span>
+                    <ul>
+                      <li>Performing assembly, QC, kraken metagenomics and aggregating statistics.</li>
+                      <li>Going from raw reads (uBAM) through to producing a phylogenetic tree.</li>
+                      <li>Working with your own data and/or public data, such as from NCBI SRA and GenBank.</li>
+                      <li>Pulling data from SRA (via SRA accession #).</li>
+                      <li>Prepare your data files for bulk upload to GenBank.</li>
+                    </ul>
+                    <a [href]="['/organizations/BroadInstitute/collections/pgs']" target="_blank" rel="noopener noreferrer">
```

Comment:
Do you intend for this to open in new tab?  I see that it does, but just asking because I don't think we typically do within the app. But if the mock has it like this, that's fine.

Reply:
Yes It was intentional. They are from different components but using the same link. 
I was using `[href]` to check for something, I will change that back, thanks for pointing that out!